debian/rules

```diff
@@ -1,13 +1,28 @@
 #!/usr/bin/make -f
 # -*- makefile -*-
 
-override_dh_auto_build:
-	md5sum -c md5sums.txt
+SOP_IMPL = sqop rsop gosop pgpainless-cli none
```

Comment:
Ah, hrmff I just noticed that `none` here does not make sense with the later wildcard. The intention was to always have a fallback value so that we would never end up running a program potentially called `armor` if SOP is empty. I think I should just move this into the foreach after the SOP_IMPL, and perhaps call it `false` or something else instead to also avoid calling some random command that might be called `none`. :D Will push an update later today.